package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cysync",
   "version": "1.0.0",
-  "description": "Cypherock CySync application",
```

Comment:
No need to change the description of the package

Reply:
resolved